tests/clients/cli/wrapper.go

```diff
@@ -27,54 +27,68 @@ import (
 	"github.com/sourcenetwork/defradb/datastore"
 	"github.com/sourcenetwork/defradb/events"
 	"github.com/sourcenetwork/defradb/http"
+	"github.com/sourcenetwork/defradb/net"
 )
 
-var _ client.DB = (*Wrapper)(nil)
+var _ client.P2P = (*Wrapper)(nil)
 
 type Wrapper struct {
-	db         client.DB
-	store      client.Store
+	node       *net.Node
 	cmd        *cliWrapper
 	handler    *http.Handler
 	httpServer *httptest.Server
 }
 
-func NewWrapper(db client.DB) *Wrapper {
-	handler := http.NewHandler(db, http.ServerOptions{})
+func NewWrapper(node *net.Node) *Wrapper {
+	handler := http.NewHandler(node, http.ServerOptions{})
 	httpServer := httptest.NewServer(handler)
 	cmd := newCliWrapper(httpServer.URL)
 
 	return &Wrapper{
-		db:         db,
-		store:      db,
+		node:       node,
 		cmd:        cmd,
 		httpServer: httpServer,
 		handler:    handler,
 	}
 }
 
+func (w *Wrapper) PeerInfo() peer.AddrInfo {
+	args := []string{"client", "p2p", "info"}
+
+	data, err := w.cmd.execute(context.Background(), args)
+	if err != nil {
+		return peer.AddrInfo{}
+	}
+	var info peer.AddrInfo
+	if err := json.Unmarshal(data, &info); err != nil {
```

Comment:
question: Here, and in the if above, we are ignoring the error. Other functions in this wrapper seem to return their errors.  How confident are you on the safety of this?  Could it not lead to the tests missing panics and errors that because of this only appear in the logs?

Reply:
I could change the API signature to return an error here. My thought process was that an empty `peer.AddrInfo` struct represents an error and any function using that empty address should return an error.

Reply:
It feels a bit wrong to change the production interface for the sake of a test wrapper (even if just to add an error return param).  Maybe we could just panic here for now (instead of returning empty)?